pkg/reloader/reloader.go

```diff
@@ -348,8 +360,8 @@ func (r *Reloader) Watch(ctx context.Context) error {
 	}
 }
 
-func normalize(logger log.Logger, inputFile, outputFile string) error {
-	b, err := os.ReadFile(inputFile)
+func (r *Reloader) normalize(input, output string) error {
```

Comment:
(nit) I preferred inputFile and outputFile as it makes it obvious that we talk about files and not data.

Reply:
Ah, an overlook on my part. Reverted.